markets/perps-market/test/integration/Account/ModifyCollateral.test.ts

```diff
@@ -187,6 +188,22 @@ describe('ModifyCollateral', () => {
         assertBn.equal(perpsBalanceAfter, perpsBalanceBefore.add(oneBTC));
       });
 
+      it('properly reflects balances after withdraw', async () => {
+        await systems()
+          .PerpsMarket.connect(trader1())
+          .modifyCollateral(accountIds[0], synthBTCMarketId, wei(-0.1).toBN());
+        const spotBalanceAfter = await synthMarkets()[0]
+          .synth()
+          .connect(trader1())
+          .balanceOf(await trader1().getAddress());
+        const perpsBalanceAfter = await synthMarkets()[0]
+          .synth()
+          .connect(trader1())
+          .balanceOf(systems().PerpsMarket.address);
+        assertBn.equal(perpsBalanceAfter, wei(0.9).toBN());
+        assertBn.equal(spotBalanceAfter, wei(0.1).toBN());
+      });
```

Comment:
i would move this into a separate test file since we will be testing a lot more withdraw calls.

and in general, move the action into a `before` and in the `it` blocks do the assertions.  and if you need to contain the logic, move all of it into its own describe, otherwise, it can potentially affect the subsequent `it` blocks.

for example, all of these `it` blocks are only adding assertions for the transaction that occured in the `before` block thats adding collateral.

Reply:
Yep that make sense. I've not split it out to failures, deposit and withdrawl